src/Solution.java

```diff
@@ -0,0 +1,11 @@
+class Solution {
+    public boolean isSameTree(TreeNode p, TreeNode q) {
+
+        if (p == null && q == null) return true;
```

Comment:
use better names for input parameters ...for example - left, right, or 
firstTree, secondTree

Reply:
this part of the code is presented by an example on leetcode